schemas/groups/electrical.json

```diff
@@ -0,0 +1,326 @@
+{
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-03/schema",
+    "id": "https://signalk.github.io/specification/schemas/groups/electrical.json#",
+    "description": "Schema describing the electrical child-object of a Vessel.",
+    "title": "electrical",
+    "properties": {
+
+        "battery": {
```

Comment:
What if there are multiple batteries? I think a sample document, even a simple one, would clear up things a lot as a pure schema is hard for humans (at least for me) to reason about. The sample document can be used to test the schema and vice versa. If you need help with this let me know.